k8s/cloud/overlays/exposed_services_nginx/cloud_ingress_grpcs.yaml

```diff
@@ -112,91 +50,28 @@ spec:
   - host: work.pixie.example.com
     http:
       paths:
-      - path: /pl.cloudapi.ArtifactTracker/
+      - path: /pl.cloudapi.(.*)
         pathType: Prefix
         backend:
           service:
             name: cloud-proxy-service
```

Comment:
Any reason you kept this as `cloud-proxy-service` rather than changing it to `api-service` like you did for `/px.cloudapi.(.*)`? 

Also, just for posterity, the `pl.cloudapi.ArtifactTracker` endpoint was kept around so that older CLIs which still excepted the endpoint to begin with `pl` would still function properly. My guess is we can actually start removing this (but we don't need to do it in this PR).

Reply:
We were doubting about this actually and IMO since we are using nginx ingress `cloud-proxy-service` seems. But `cloud-proxy-service` has some additional configurations related to caching and compression, so that's why we kept it around.

Will remove `pl.*` http paths.

Reply:
@aimichelle shell I route all the traffic to `api-service` and ignore the `cloud-proxy-service`?